test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerFhirStorageTestHelper.cs

```diff
@@ -51,7 +51,7 @@ public class SqlServerFhirStorageTestHelper : IFhirStorageTestHelper, ISqlServer
         public async Task CreateAndInitializeDatabase(string databaseName, int maximumSupportedSchemaVersion, bool forceIncrementalSchemaUpgrade, SchemaInitializer schemaInitializer = null, CancellationToken cancellationToken = default)
         {
```

Comment:
Ah, so we _were_ passing in `maximumSupportedSchemaVersion` but only using it in the initialization of the `SqlServerFhirModel` 🤔